config/k8s-etcd.txt

```diff
@@ -0,0 +1,105 @@
+# [This file is part of the zabbix-agent-osso package]
+k8s-etcd
+===========
+
+K8s-etcd monitors an in-cluster etcd setup. the status of the etcd is written to a configmap with the following (example) pod.
```

Comment:
Kan mooi tot 72 à 80 columns uitgelijnd. En "the status" is na een punt, dus gerne met hoofdletter.